### ISSUE TITLE
[BUG] archaeologybanners has a broken icon

### ISSUE THREAD

**Issue by SammyForReal** [Darkhax-Minecraft/Archaeology-Banners#4]
### Minecraft Version

1.20.5

### Mod Version

v8.0.2

### Mod Loader

Fabric

### What environment are you running the mod in?

Client

### Issue Description

Hello! While using the ModMenu, I always get the warning in my logs saying that this mod has a "broken icon".
This is the full log message:
```
[Render thread/WARN]: Warning! Mod archaeologybanners has a broken icon, loading default icon
[00:39:14] [Render thread/ERROR]: Mod icon must be a square for icon source mr_blood_moon: blood-moon_pack.png
java.lang.IllegalStateException: Must be square icon
	at org.apache.commons.lang3.Validate.validState(Validate.java:773) ~[commons-lang3-3.13.0.jar:?]
	at com.terraformersmc.modmenu.util.mod.fabric.FabricIconHandler.createIcon(FabricIconHandler.java:36) ~[modmenu-9.2.0-beta.2.jar:?]
	at com.terraformersmc.modmenu.util.mod.fabric.FabricMod.getIcon(FabricMod.java:163) ~[modmenu-9.2.0-beta.2.jar:?]
	at com.terraformersmc.modmenu.gui.widget.entries.ModListEntry.getIconTexture(ModListEntry.java:131) ~[modmenu-9.2.0-beta.2.jar:?]
	at com.terraformersmc.modmenu.gui.widget.entries.ModListEntry.method_25343(ModListEntry.java:60) ~[modmenu-9.2.0-beta.2.jar:?]
	at com.terraformersmc.modmenu.gui.widget.ModListWidget.method_25311(ModListWidget.java:239) ~[modmenu-9.2.0-beta.2.jar:?]
	at net.minecraft.class_350.method_48579(class_350.java:184) ~[client-intermediary.jar:?]
	at net.minecraft.class_339.method_25394(class_339.java:66) ~[client-intermediary.jar:?]
	at com.terraformersmc.modmenu.gui.ModsScreen.method_25394(ModsScreen.java:303) ~[modmenu-9.2.0-beta.2.jar:?]
	at net.minecraft.class_437.method_47413(class_437.java:110) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.mixinextras$bridge$method_47413$187(class_757.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.wrapOperation$bhh000$fabric-screen-api-v1$onRenderScreen(class_757.java:2611) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:931) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1327) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[minecraft-1.20.4-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.11.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
```

**Comment by Darkhax** [comment]
Thanks for letting me know, I will hopefully fix this soon.

**Comment by Darkhax** [comment]
This issue should now be fixed on 1.20.1 and 1.20.4. You mentioned 1.20.5 but that version is not supported at all yet. It's also worth noting that most of the error you pasted is for the "mr blood moon" mod and not archaeology banners.
